Ogury/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+## Changelog
+  * 5.0.7.1
+    * Initial commit
+    * Add support for banner, interstitial and rewarded ads.
+    * This version of the adapters has been certified with Ogury SDK 5.0.7 and MoPub 5.16.4.
```

Comment:
**question** (non-blocking): I'm not sure we should be the one adding this line.

By looking at the commits that changes the change logs, it seems that its the people at Twitter that manages them, not the adapter creators.

Reply:
Fixed in ecdb97582edd887a4ba69ae77ba6a6bd05a8d402.